### ISSUE TITLE
New Lines after blank entry aren't considered.

### ISSUE THREAD

**Issue by mahonb** [testdouble/clojurescript.csv#13]
When you have a file that has blank fields at the end of a row, the new line character for those entries is not considered.
For the following file
```
a,b,c
1,1,1
2,,
3,,
```

The output of `read-csv` currently is:
```
[[ "a", "b", "c"]
 ["1", "1", "1"]
 ["2", "", "", "\n3", "", "\n"]]
```

**Comment by rosston** [comment]
That sure looks like a bug! Pull requests are welcome if you are interested in fixing it. Otherwise, I will probably take a look at fixing this in the next couple weeks.

**Comment by rosston** [comment]
Apologies for the long delay here. This is now fixed in 0.4.4. I discovered another bug (https://github.com/testdouble/clojurescript.csv/issues/17) while fixing this one, but I'll tackle that separately.
